### PR TITLE
User connections issue

### PR DESCRIPTION
**Pull request by AndrePu**

### CODE REVIEW

File: Cooper/Repository/UsersConnection/UsersConnectionRepository.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            mapper = new ModelsMapper();
+        }
+        public List<User> GetSpecifiedTypeUsersList(long userId, ConnectionType specifiedType)
+        {
+            List<UserDb> users = userConnectionsDAO.GetSpecifiedTypeUsersList(userId, specifiedType);
+            List<User> users_newTyped = new List<User>();
+
+            if (users != null)
+            {
+                foreach (var user in users)
```
Comment (Hunroll) [inline Cooper/Repository/UsersConnection/UsersConnectionRepository.cs]: Can be replaced with
users_newTyped = users.Select(mapper.Map).ToList();

File: Cooper/Controllers/UsersConnectionController.cs [hunk 2]
```diff
@@ -71,45 +72,45 @@ public IActionResult Subscribe(long userId)
...
-        public IActionResult Ban(long userId)
+        public IActionResult Ban(long bannedUserId)
         {
             if (!ModelState.IsValid)
             {
                 return BadRequest(ModelState);
             }
 
-            string subscriberToken = Request.Headers["Authorization"].ToString().Replace("Bearer ", "");
+            string userToken = Request.Headers["Authorization"].ToString().Replace("Bearer ", "");
```
Comment (Hunroll) [inline Cooper/Controllers/UsersConnectionController.cs]: I wanted to propose creating an extension method that will return the user from request, but I have already found it in code)))
So just use it) or add the similar one for a token

File: Cooper/Repository/UsersConnection/UsersConnectionRepository.cs (hunk 1 above)
Comment (AndrePu) [reply Cooper/Repository/UsersConnection/UsersConnectionRepository.cs]: Yeep)

File: Cooper/Controllers/UsersConnectionController.cs (hunk 2 above)
Comment (AndrePu) [reply Cooper/Controllers/UsersConnectionController.cs]: Just added extraa method in AuthExtension for taking token, so logic for creating UsersConnection object by appropriate service is saved

**Pull request merged**